internal/services/recoveryservices/site_recovery_replicated_vm_resource.go

```diff
@@ -678,24 +700,131 @@ func resourceSiteRecoveryReplicatedItemRead(d *pluginsdk.ResourceData, meta inte
 	d.Set("source_recovery_protection_container_name", id.ReplicationProtectionContainerName)
 
 	if prop := model.Properties; prop != nil {
-		d.Set("target_recovery_fabric_id", prop.RecoveryFabricId)
-		d.Set("recovery_replication_policy_id", prop.PolicyId)
-		d.Set("target_recovery_protection_container_id", prop.RecoveryContainerId)
+		recoveryFabricId := ""
+		if fabricId := pointer.From(prop.RecoveryFabricId); fabricId != "" {
+			parsedFabricId, err := replicationfabrics.ParseReplicationFabricIDInsensitively(fabricId)
+			if err != nil {
+				return fmt.Errorf("parsing recovery_fabric_id %s: %+v", fabricId, err)
```

Comment:
The ID parsers already return detailed error messages so can we simplify all the error messages to 
```suggestion
				return err
```